### ISSUE TITLE
Failed to sign the game, please ask developer for help.

### ISSUE THREAD

**Issue by Skidaddlerow** [ZaneYork/SMAPI-Android-Installer#210]
![Screenshot_2022-05-01-20-44-02-800_com zane smapiinstaller](https://user-images.githubusercontent.com/89689239/166146693-ccb21c1f-114a-4162-bda9-5b102013d0fd.jpg)

**Comment by shbzmk** [comment]
Yes same problem for me please developer fix this problem imidiatly or tell me some solution

**Comment by selimkarabey** [comment]
![Uploading Screenshot_2022-06-17-00-30-04-399_com.zane.smapiinstaller.jpg…]()
